content/docs/instrumenting/exporters.md

```diff
@@ -73,6 +73,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NVIDIA GPU exporter](https://github.com/mindprince/nvidia_gpu_prometheus_exporter)
    * [ProSAFE exporter](https://github.com/dalance/prosafe_exporter)
    * [Ubiquiti UniFi exporter](https://github.com/mdlayher/unifi_exporter)
+   * [Windows/system metrics exporter](https://github.com/prometheus-community/windows_exporter)
```

Comment:
Just Windows will do. The system metrics for the node exporter is due to the ambiguity of "node" which has caused confusion.